main.tf

```diff
@@ -155,6 +185,38 @@ resource "aws_db_instance" "db_excluding_name" {
   tags = "${merge(var.tags, map("Name", format("%s-%s", var.environment, var.name)), map("Env", var.environment))}"
 }
 
+# The database instance itself with read replica
+
+resource "aws_db_instance" "db_excluding_name_with_read_replica" {
```

Comment:
This could be named a little better e.g. `db_read_replica` or something like that